src/tejashree/NoOfCharInString.java

```diff
@@ -0,0 +1,42 @@
+package tejashree;
+import java.util.Scanner;
+
+class NoOfCharInString {
+
+	void findFrequency(String word, char ch) {
+		int cnt = 0;
+		for (int index = 0; index < word.length(); index++) {
+			if (word.charAt(index) == ch) {
+				cnt++;
+			}
+		}
+		if(cnt>0) 
+			System.out.println("char" + "'" + ch + "'" + "is repeated " + cnt + " times");
+		else
+			System.out.println("character is not present in a string");
+	}
+	public static void main(String[] args) {
+		NoOfCharInString noOfCharInString = new NoOfCharInString();
+		Scanner sc = new Scanner(System.in);
+		System.out.println("Enter a string");
+		String userInput = sc.next();
+		
+		System.out.println("Enter character");
+		String temp=sc.next();
+		int count=1;
+		while(temp.length()!=1 && count<3) {
```

Comment:
Why we are checking length()!=1 it should be if length()==0 then user not given input and we want user to give input here. condition need to be update

Reply:
assume user pass ABCD in tmep then still you are asking user to give input